### PR TITLE
Adds new custom hand-wired hot-swap mono block RP2040 board nyaaaah54_rp2040 to upstream

### PR DESCRIPTION
**Pull request by jkaunert**
## Description

Adds my new custom handwired hotswap mono block keyboard 'nyaaaah54_rp2040' and default keymap/layout to upstream

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/philoandgunge/nyaaaah54_rp2040/nyaaaah54_rp2040.c
Comment (zvecr) [inline keyboards/philoandgunge/nyaaaah54_rp2040/nyaaaah54_rp2040.c:1]: This file should be removed.

File: keyboards/philoandgunge/nyaaaah54_rp2040/nyaaaah54_rp2040.h
Comment (zvecr) [inline keyboards/philoandgunge/nyaaaah54_rp2040/nyaaaah54_rp2040.h:1]: This file should be removed.

File: keyboards/philoandgunge/nyaaaah54_rp2040/info.json [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    "bootmagic": true,
+    "extrakey": true,
+    "mousekey": true,
+    "nkro": true
+  },
+  "ws2812": {
+    "pin": "GP29",
+    "driver": "vendor"
+  },
+  "debounce": 5,
```
Comment (zvecr) [inline keyboards/philoandgunge/nyaaaah54_rp2040/info.json:50]:
```suggestion
```

File: keyboards/philoandgunge/nyaaaah54_rp2040/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# nyaaaah54_rp2040
+
+![nyaaaah54_rp2040](https://i.imgur.com/nyaaaah54_rp2040.jpg)
+
+A trashy, handwired hotswap-ish monoblock ergo. Made from salvaged & upcycled parts and materials from the all-knowing trash heap in the back alleyway.
```
Comment (zvecr) [inline keyboards/philoandgunge/nyaaaah54_rp2040/README.md:5]: > handwired

Project **must** be relocated be within the `keyboards/handwired` directory.

File: keyboards/philoandgunge/nyaaaah54_rp2040/keymaps/default/keymap.json [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+      "KC_TRNS",
+      "KC_TRNS",
+      "KC_TRNS",
+      "KC_TRNS",
+      "KC_TRNS",
+      "KC_TRNS",
+      "KC_TRNS"
+    ]
+  ],
+  "author": "gunge the great and glorious",
```
Comment (zvecr) [inline keyboards/philoandgunge/nyaaaah54_rp2040/keymaps/default/keymap.json:121]:
```suggestion
  "author": "jkaunert",
```

File: keyboards/philoandgunge/nyaaaah54_rp2040/rules.mk [hunk 4]
```diff
@@ -0,0 +1 @@
+BLUETOOTH = AdafruitBLE
```
Comment (zvecr) [inline keyboards/philoandgunge/nyaaaah54_rp2040/rules.mk:1]: This option is not valid, and its replacement can be specified in info.json
```suggestion
# This file intentionally left blank
```

File: keyboards/philoandgunge/nyaaaah54_rp2040/config.h
Comment (zvecr) [inline keyboards/philoandgunge/nyaaaah54_rp2040/config.h:1]: This file should be removed.

File: keyboards/philoandgunge/nyaaaah54_rp2040/config.h [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#define BACKLIGHT_PIN GP22
+/* number of backlight levels */
+
+#ifdef BACKLIGHT_PIN
+#define BACKLIGHT_LEVELS 6
+#endif
```
Comment (zvecr) [inline keyboards/philoandgunge/nyaaaah54_rp2040/config.h:25]: Backlight config (which should be in info.json) but feature not configured.

File: keyboards/philoandgunge/nyaaaah54_rp2040/README.md
Comment (zvecr) [inline keyboards/philoandgunge/nyaaaah54_rp2040/README.md:1]: Filename should be lowercase.

File: keyboards/philoandgunge/nyaaaah54_rp2040/info.json [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+  "keyboard_name": "nyaaaah54_rp2040",
+  "manufacturer": "0xPHGU",
+  "url": "https://philoandgunge.com",
+  "maintainer": "jkaunert",
+  "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+   },
+  "matrix_pins": {
```
Comment (zvecr) [inline keyboards/philoandgunge/nyaaaah54_rp2040/info.json:11]: Mixed indentation, should be formatted with `qmk format-json`

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/philoandgunge/nyaaaah54_rp2040/README.md (hunk 2 above)
Comment (Anutrix) [reply keyboards/philoandgunge/nyaaaah54_rp2040/README.md:5]: Link doesn't work either

File: keyboards/philoandgunge/nyaaaah54_rp2040/info.json (hunk 6 above)
Comment (jkaunert) [reply keyboards/philoandgunge/nyaaaah54_rp2040/info.json]: all resolved. thanks for reviewing!

Comment (zvecr) [reply keyboards/philoandgunge/nyaaaah54_rp2040/info.json]: Still needs formatting.

Comment (jkaunert) [reply keyboards/philoandgunge/nyaaaah54_rp2040/info.json]: reformatted again from terminal inside vscode, hopefully that corrects the mixed indents.

File: keyboards/handwired/philoandgunge/nyaaaah54_rp2040/info.json [hunk 7]
```diff
@@ -0,0 +1,543 @@
...
+  "usb": {
+    "device_version": "1.0.0",
+    "pid": "0x0000",
+    "vid": "0xFEED"
+  },
+  "ws2812": {
+    "driver": "vendor",
+    "pin": "GP29"
+  },
+  "layouts": {
```
Comment (zvecr) [inline keyboards/handwired/philoandgunge/nyaaaah54_rp2040/info.json:51]: The formatter does not format like this, please use the command previously mentioned.

File: keyboards/handwired/philoandgunge/nyaaaah54_rp2040/info.json [hunk 8]
```diff
@@ -0,0 +1,543 @@
+{
+  "manufacturer": "0xPHGU",
+  "keyboard_name": "nyaaaah54_rp2040",
+  "maintainer": "jkaunert",
+  "bootloader": "rp2040",
+  "debounce": 5,
```
Comment (zvecr) [inline keyboards/handwired/philoandgunge/nyaaaah54_rp2040/info.json:6]:
```suggestion
```

File: keyboards/handwired/philoandgunge/nyaaaah54_rp2040/readme.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+
+A trashy, handwired hotswap-ish monoblock ergo. Made from salvaged & upcycled parts and materials from the all-knowing trash heap in the back alleyway.
+
+* Keyboard Maintainer: [jkaunert](https://github.com/jkaunert)
+* Hardware Supported: Handwired, rp_2040W(support for more boards, BLE in future revisions), gateron hot-swap sockets
+* Hardware Availability: [Build log/data coming soon](https://philoandgunge.com/handwired/builds/nyaaaah54_rp2040)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make philoandgunge/nyaaaah54_rp2040:default
```
Comment (zvecr) [inline keyboards/handwired/philoandgunge/nyaaaah54_rp2040/readme.md:13]:
```suggestion
    make handwired/philoandgunge/nyaaaah54_rp2040:default
```

File: keyboards/handwired/philoandgunge/nyaaaah54_rp2040/readme.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+
+Flashing example for this keyboard:
+
+    make philoandgunge/nyaaaah54_rp2040:default:flash
```
Comment (zvecr) [inline keyboards/handwired/philoandgunge/nyaaaah54_rp2040/readme.md:17]:
```suggestion
    make handwired/philoandgunge/nyaaaah54_rp2040:default:flash
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/handwired/philoandgunge/nyaaaah54_rp2040/info.json [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+    "qmk": {
+        "tap_keycode_delay": 10
+    },
+    "url": "https://philoandgunge.com",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "ws2812": {
```
Comment (drashna) [inline keyboards/handwired/philoandgunge/nyaaaah54_rp2040/info.json:27]: but no rgb is enabled, and number of leds isn't defined?

File: keyboards/handwired/philoandgunge/nyaaaah54_rp2040/info.json [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+        "driver": "vendor",
+        "pin": "GP29"
+    },
```
Comment (drashna) [inline keyboards/handwired/philoandgunge/nyaaaah54_rp2040/info.json:30]:
```suggestion
```

File: keyboards/handwired/philoandgunge/nyaaaah54_rp2040/readme.md [hunk 13]
```diff
@@ -0,0 +1,25 @@
+# nyaaaah54_rp2040
+
+![nyaaaah54_rp2040](https://imgur.com/a/vgqw9SO)
```
Comment (drashna) [inline keyboards/handwired/philoandgunge/nyaaaah54_rp2040/readme.md:3]:
```suggestion
![nyaaaah54_rp2040](https://i.imgur.com/Kig3OeGh.jpeg)
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**